build_scripts/build_runner.py

```diff
@@ -866,9 +868,17 @@ def _strip_bins(self):
                                    debug_file])
                 ])
 
+                check_binary_command = f'file {orig_file} | grep ELF'
+
                 for command in strip_commands.values():
                     err, out = cmd_exec(command, shell=False, log=self.log, verbose=False)
                     if err:
+                        # Not strip file if it is not binary
+                        return_code, out_check_binary = cmd_exec(check_binary_command, shell=True, log=self.log,
+                                                                 verbose=False)
+                        if return_code:
+                            self.log.warning(f"File {orig_file} not binary")
+                            break
```

Comment:
Do we really want to break stripping? Does it a showstopper?

Reply:
If file is not binary, we stop implementing commands in strip_commands dict and show only one log warning message for current string